Vulnerability.Scanner.Api/Program.cs

```diff
@@ -16,11 +16,17 @@ public static class Program
         public static void Main(string[] args)
         {
             var host = CreateWebHostBuilder(args)
-                .ConfigureAppConfiguration((hostingContext, config) => config.AddCommandLine(args))
+                .ConfigureAppConfiguration((_, config) => config
+                    .AddCommandLine(args)
+                    .AddEnvironmentVariables())
                 .Build();
 
-            if(!string.IsNullOrEmpty(host.Services.GetService<IOptions<GeneralOptions>>().Value.ConnectionString))
+            var connectionStringIfAny = host.Services.GetService<IOptions<GeneralOptions>>().Value.ConnectionString;
+
+            if(!string.IsNullOrEmpty(connectionStringIfAny))
             {
+                var passwordRegex = new Regex("Password=.*?;");
```

Comment:
Not sure but can connection string parameters be lowercase?

Reply:
Don't know either, added ignore case for it.